### PR TITLE
Country selection

### PR DESCRIPTION
**Pull request by MattiaSilvestri**
Changes:
- user can choose between automatic geolocation and manual country selection;
- minimal cli;
- help message;
- flag for getting country list in alphabetical order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,10 @@ Simple tool to monitor your personal computer power consumption and emissions.
 ### Useful Links
 - [Electricity Maps](https://app.electricitymaps.com/map)
 - [Co2Signal API](https://www.co2signal.com/)
+
```
Comment (FrnRmn) [inline README.md]: next time I would do this changes outside the branch, just for future commits

File: src/main.py [hunk 2]
```diff
@@ -1,21 +1,69 @@
...
+        cpu_usage = em.get_cpu_usage(10) #it seems too low to be true!
+        print("- Current CPU: {0} \n- CPU usage during last 10s: {1}%".format(cpu_info, cpu_usage))
+    except KeyError:
+        print('Sorry, your country is not present in the Electricity Map ' +
+              'database, it is therefore not possible to retrieve CO2 emissions data.')
+
+
+args = cli.get_args()
+
+if args.list:
```
Comment (FrnRmn) [inline src/main.py:8]: I would extract the next rows as a function outside main.py

Comment (FrnRmn) [reply src/main.py:8]: possible?

**Review by FrnRmn — COMMENTED** [review]
Looks good in general, thanks

File: src/energymonitor/exceptions.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+"""Define custom exceptions"""
```
Comment (FrnRmn) [inline src/energymonitor/exceptions.py]: to be removed

**Review by FrnRmn — CHANGES_REQUESTED** [review]
I think this is the only necessary change, thanks

File: README.md (hunk 1 above)
Comment (MattiaSilvestri) [reply README.md]: Not sure I got it, you mean to not update the README directly in dev or main?

**Comment by MattiaSilvestri** [comment]
Made some changes:
- removed the exceptions module;
- created the "core" module which will host the scripts handling the core functionalities of the apps. I moved the main there and created a new "cpuCO2" script which will be used only for launching.

File: README.md (hunk 1 above)
Comment (FrnRmn) [reply README.md]: No no, just that the README to-do list seems outside the scope of this"country_selection" branch

**Comment by FrnRmn** [comment]
so many changes, basically a new pr haha :D
I might need a call to clarify a few things

**Comment by MattiaSilvestri** [comment]
Yeah sorry ahah, I tried to add the things you asked, but I'm still not very sure about the core package, I just didn't know how to better separate the two parts of the main (user arguments handling and main() funciton), but yeah we can discuss it

**Pull request merged**